lib/loadRepoConfig.js

```diff
@@ -85,7 +87,8 @@ loadRepoConfig._getTemplate = function (url, headers) {
             json: true
         })
         .then(function (response) {
-            return response.content;
+            return Buffer.from(response.content, response.encoding)
```

Comment:
I'm confused why this is needed.  I noticed `json:true` is used in the above get call, is this data JSON?

Reply:
`response.content` is an encoded string - https://developer.github.com/v3/repos/contents/#get-contents

Reply:
Thanks.